src/main/java/org/apache/commons/imaging/formats/jpeg/JpegConstants.java

```diff
@@ -125,7 +136,9 @@
                     SOF5_MARKER, SOF6_MARKER, SOF7_MARKER, SOF8_MARKER, SOF9_MARKER,
                     SOF10_MARKER, SOF11_MARKER, DAC_MARKER, SOF13_MARKER,
                     SOF14_MARKER, SOF15_MARKER, EOI_MARKER, SOS_MARKER, DQT_MARKER,
-                    DNL_MARKER, COM_MARKER));
+                    DNL_MARKER, COM_MARKER,
```

Comment:
There will be a minor conflict here when rebase/merging it, but can be easily resolved by whoever merges it.